### ISSUE TITLE
[Topic] WordPress plugins and accessibility

### ISSUE THREAD

**Issue by rianrietveld** [wpaccessibility/wp-a11y-docs#165]
URL: https://wpaccessibility.org/docs/topics/plugins/
Code: https://github.com/wpaccessibility/wp-a11y-docs/blob/main/docs/topics/plugins/index.md

Please read [How to contribute](https://wpaccessibility.org/docs/contribute/) before you start.

## Tasks
The accessibility content should be moved to the topics, this page should only contain practical info.
Which info that will be is open for discussion. Info for plugin developers and for people choosing a plugin?

Avoid duplicate content. The accessibility content should be reviewed for modern code and the links adjusted and divided between the topics.  

## Discussion
Comment by @AnneliesDT 
> It’s somewhat confusing what the difference is between WP Accessibility plugins and Accessible WP Plugins (there are none so far). If there are no Accessible WP Plugins, maybe leave this page out for now? Or add more content on how to build accessible WP plugins, same as to how to build accessible themes (which has more info, but still very little).

Comment Rian: 
> I think it’s important to communicate that there is no accessibility-ready tag for plugins. The page should stay, the text could be more clear.
